src/boost-api/Endpoint_Handlers.cpp

```diff
@@ -46,6 +46,27 @@ namespace api {
         return json_data;
     }
 
+    boost::json::array convert_trajectory_to_json(const std::vector<data_structures::Trajectory>& trajectory){
+        boost::json::array response_trajectory{};
+        boost::json::object json_traj{};
+        json_traj["id"] = trajectory[0].id;
+
+        boost::json::array json_locations{};
+        for (const auto &loc: trajectory[0].locations) {
+            boost::json::object json_location{};
+            json_location["order"] = loc.order;
+            json_location["timestamp"] = loc.timestamp;
+            json_location["longitude"] = loc.longitude;
+            json_location["latitude"] = loc.latitude;
+            json_locations.push_back(std::move(json_location));
+        }
+
+        json_traj["locations"] = std::move(json_locations);
+        response_trajectory.push_back(std::move(json_traj));
+
+        return response_trajectory;
```

Comment:
Return it `boost::json::object` i stedet for et array, hvis det er muligt.

Reply:
done